nixos/modules/services/networking/firewall.nix

```diff
@@ -177,6 +177,23 @@ let
       ) cfg.allowedUDPPortRanges
     }
 
+    # Accept packets on the extra allowed host+port combination.
+    ${concatMapStrings (allowed:
+        concatMapStrings (addressFamily:
+        concatMapStrings (protocol:
+          (if addressFamily == "IPv4" then "iptables -A nixos-fw "
+           else "ip6tables -A nixos-fw ")
```

Comment:
Could you please add the `-w` option to `iptables` and `ip6tables` (just to be safe :smile:)?